src/app/tests/suites/certification/Test_TC_OO_2_3.yaml

```diff
@@ -817,8 +821,8 @@ tests:
       PICS: OO.S.A4001 && OO.S.C42.Rsp
       response:
           constraints:
-              minValue: 170
-              maxValue: 230
+              minValue: 200 - PIXIT.OO.MaxCommunicationTurnaround
```

Comment:
In 19b the `OnTime` countdown starts.
In first 19c `readAttribute` of `OnOff` is performed that can take up to `PIXIT.OO.MaxCommunicationTurnaround`.
In second 19c `readAttribute` of `OnTime` is performed that can take up to `PIXIT.OO.MaxCommunicationTurnaround`.
Additionally I think that 15% tolerance applies here, it elapsed 10 seconds since the countdown starts, 15% of 100 is 15, so in total `185`.
So in total I think it should be `185 - 2 * PIXIT.OO.MaxCommunicationTurnaround`.

Reply:
Updated